api/src/main/resources/workbench.yaml

```diff
@@ -1100,6 +1100,22 @@ paths:
           schema:
             $ref: "#/definitions/Cohort"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/{cohortId}/duplicate:
```

Comment:
I think I prefer keeping it as Duplicate so it follows the same naming convention as in the UI. It might not be consistent with the other `clone` methods but IMO its easier for a developer to find the right code to change if the code matches the UI.

If we change the language back to clone, its easy enough to change. I can actually change the existing `cloneX` methods to be `duplicateX` if you're OK with it.

Reply:
Sure feel free to file a ticket + handle in a follow-up PR. My view is that we shouldn't be chasing product/UI changes with our API (it's a moving target), but if you feel strongly about the naming in this case I'm not going to object - as long as we reach consistency.